tests/lib/__init__.py

```diff
@@ -358,6 +339,14 @@ def _ignore_file(self, fn):
             result = super(PipTestEnvironment, self)._ignore_file(fn)
         return result
 
+    def _find_traverse(self, path, result):
```

Comment:
A comment mentioning why this underscore method override is done, would be nice.